### ISSUE TITLE
Add new header with multiple navbars

### ISSUE THREAD

**Issue by chrismlusk** [chrismlusk/accuraty-templates#26]
Default Bootstrap has one `.navbar` that you can easily add multiple `.nav` sections to, but they end up all on the same row. Many clients want/need multiple navs in different locations in the header, though: eyebrow menu, secondary nav, etc.

For example, see https://github.com/Accuraty/PDEG/pull/6.

Let's add a template to the `includes/` directory to handle this.

**Comment by Accuraty** [comment]
Any chance you already have enough of a pattern so the "Manage" nav/menu could also become a default? Keep in mind, that DNN 9+ we no longer name it "Manage" - from now on it is the "SITE" menu - and it will almost always exist on all our DNN projects will have a least  a few of these semi-standard pages:
### SITE
> Manage Users
> Manage Articles and Events
> Manage Publications
> Help (How-To, Reminders):
>> Filenames
>> Images
>> 5 Easy SEO Tips

**Comment by chrismlusk** [comment]
I kind of hate the name "Site" for this. I see that "Manage" doesn't work because the section encompasses more than just site management, but don't you think "Site" is too vague?

**Comment by Accuraty** [comment]
I agree and it doesn't make sense to me. However, when you train on it and look at it from the client's point of view... "SITE", as in "my site" seems to make the most sense. Especially when you realize the workers at the Content manager or Site manager level have no idea there is an Admin or Super user level. So the menu is named to let them know that this is where all the "my site" stuff is.
 
THAT HAVING BEEN SAID - I am open to another name. Just waiting until someone floats the right one in front of me. Waiting...

**Comment by chrismlusk** [comment]
Well... what about "My Site" instead??

**Comment by Accuraty** [comment]
Well... what about "Accuraty" instead??

**Comment by Accuraty** [comment]
While we are on the topic... I think it would also be nice to have both the ISAA full-expando and PubsPlus sidebar-pushout menus as 'easy-to-activate' options. :)
